installparty.sh

```diff
@@ -49,10 +49,12 @@ source ~/.bashrc
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 
-nvm -v 
+nvm -v
 nvm install stable
 nvm use stable
 
```

Comment:
https://raw.githubusercontent.com/{username}/{reponame}/{branchname}/{filename} 

@kangarang for reference